### PR TITLE
Pre release tests

### LINKED ISSUE #8: Make pre-release-tests.yml file
During the [pre-release week](https://github.com/meilisearch/integration-guides/blob/master/guides/pre-release-week.md) (concerning only the Meili team, don't bother with the details) I need a CI running against the latest <ins>RC</ins> of MeiliSearch (and not the `latest` image). This RC is
Since the CIs of this package run in a Dart container, I don't know how to specify a dynamic variable (get by a script) to define which MeiliSearch version I would like to start.

See the example in the Ruby SDK CI: 
https://github.com/meilisearch/meilisearch-ruby/blob/2ce58bdda136c1b672fbbbcdefb41f49a69d85c4/.github/workflows/pre-release-tests.yml#L23-L26

The `pre-release-tests.yml` file in this package would start the same:

```yml
# Testing the code base against the MeiliSearch pre-releases
name: Pre-Release Tests

# Will only run for PRs and pushes to bump-meilisearch-v*
on:
  push:
    branches:
      - bump-meilisearch-v*
  pull_request:
    branches:
      - bump-meilisearch-v*

jobs:
  integration-tests:
    timeout-minutes: 10
    runs-on: ubuntu-latest
    strategy:
      matrix:
        version: ['2.7', '2.8']
    name: integration-tests-against-rc (dart ${{ matrix.version }})
    
   ...
```

I just currently don't know how to run the latest RC of MeiliSearch in this GitHub Action.

EDIT: Oh maybe use the linux-amd64 assets link to download, unzip and run meilisearch with it: https://github.com/meilisearch/MeiliSearch/releases/download/v0.17.0rc0/meilisearch-macos-amd64

### PR DESCRIPTION
**Pull request by TheMisir**
Fixes #8

### CODE REVIEW

**Comment by TheMisir** [comment]
bors test

File: example/pubspec.lock [hunk 1]
```diff
@@ -35,7 +35,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "0.1.0+1"
+    version: "0.1.1+1"
```
Comment (curquiza) [inline example/pubspec.lock:38]: Why do you upgrade the version for this kind of update? We don't release for this kind of change that does not impact the user, right?

**Review by curquiza — COMMENTED** [review]
Awesome!! Smart idea! 👍

**Comment by curquiza** [comment]
bors try

**Comment by curquiza** [comment]
bors-

**Comment by curquiza** [comment]
bors try

File: example/pubspec.lock (hunk 1 above)
Comment (TheMisir) [reply example/pubspec.lock:38]: The change was made automatically by my IDE/editor. The update was made to example project not the package. Also pubspec.lock file is used to synchronize package versions between developers who are working on the same project. Don't worry it doesn't impacts users. :)

Comment (curquiza) [reply example/pubspec.lock:38]: Ok, I don't' really understand how releases work for pub.dev and in the dart community! 😅 I will probably ask you questions directly on Slack 😄

**Review by curquiza — APPROVED** [review]
bors merge

File: example/pubspec.lock (hunk 1 above)
Comment (TheMisir) [reply example/pubspec.lock:38]: Ok, the changed file is lockfile used by dart's package manager (pub). It's same as package-lock.json / yarn.lock files in node.js / npm.

Comment (curquiza) [reply example/pubspec.lock:38]: Oh that's the file in the `example` folder! You told me but I did not get it! haha
Sorry for being a 🐌 today.

**Pull request merged**